src/ChannelServer/Database/Account.cs

```diff
@@ -86,6 +87,17 @@ public Account()
 			this.Variables = new Variables();
 			_chatMacros = new List<ChatMacro>();
 			_revealedMaps = new Dictionary<int, RevealedMap>();
+
+			// Initialize with the default set of macros.
+			var macroData = ChannelServer.Instance.Data.ChatMacroDb
+				.Where(x => x.Id <= 10) // maximum number of chat macros allowed.
```

Comment:
While this will probably never fail, I have a bad feeling when I look at it, because *theoretically* the database could suddenly include a 0 id, or even negative ones, be it through a bug, or a user error, or whatever. It would be safer to get all, order them, and add them until you've reached a max of 10.

And what about the 0 poses, should they actually be added? Like I said, I only get 7 macros when I log into iToS.